cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js

```diff
@@ -385,14 +385,29 @@ export default class ColumnActionsDropdown extends Component {
   render() {
     return (
       <span className="column-actions-dropdown-container" ref={(ref) => (this.popover = ref)}>
-        <span
+        <button
           className={classnames('fa fa-caret-down', {
             expanded: this.state.dropdownOpen,
           })}
           onClick={this.toggleDropdown}
           id={`dataprep-action-${this.dropdownId}`}
         />
-
+        <UncontrolledTooltip
+          target={`dataprep-action-${this.dropdownId}`}
+          placement="top"
+          modifiers={{
+            flip: {
+              enabled: false,
+              boundariesElement: document.querySelector('.dataprephome-wrapper'),
+            },
+            // FIXME: This offset is not being applied because the popover overlaps with its boundary.
```

Comment:
@yukiej nit: Can you file a JIRA to fix the overlap. If we don't have a JIRA with FIXMEs we won't get to fix them

Reply:
@ajainarayanan Made JIRA and added issue number in the comment!